pkg/cloud/services/eks/roles.go

```diff
@@ -52,6 +54,22 @@ func FargateRolePolicies() []string {
 	}
 }
 
+// NodegroupRolePoliciesAWSUSGov gives the policies required for a nodegroup role.
+func NodegroupRolePoliciesAWSUSGov() []string {
```

Comment:
we should refactor and reuse  NodegroupRolePolicies 

as they are exactly same and only difference is partition 

maybe a common function that takes partition as argument

Reply:
Have optimised this code